src/markers/MarkerArrayClient.js

```diff
@@ -100,3 +96,16 @@ ROS3D.MarkerArrayClient.prototype.unsubscribe = function(){
     this.rosTopic.unsubscribe();
   }
 };
+
+ROS3D.MarkerClient.prototype.removeMarker = function(key) {
```

Comment:
```suggestion
ROS3D.MarkerArrayClient.prototype.removeMarker = function(key) {
```
This needs to be changed to `MarkerArrayClient` (copy/paste mistake)